packages/manager/apps/dedicated/client/app/components/dedicated-cloud/taskPollProgress/taskPollProgress.component.js

```diff
@@ -0,0 +1,13 @@
+import controller from './taskPollProgress.controller';
+import template from './taskPollProgress.html';
+
+export default {
+  bindings: {
+    dedicatedCloud: '<',
+    loaders: '=',
+    productId: '<',
+    task: '=',
```

Comment:
i guess one-time binding `<`, instead of two-time binding `=` is enough, no ?

Reply:
We need return of module taskPollProgress.